src/test/resources/org/prebid/server/it/openrtb2/improvedigital/test-auction-improvedigital-request-2.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
In one of our previous request, we updated our It test to be as much simpler as possible, so pls remove test-auction-request-2, and related files